src/checkers/english.rs

```diff
@@ -0,0 +1,24 @@
+// import storage
+use crate::storage;
+
+// given an input, check every item in the array and return true if any of them match
+pub fn check_english(input: &str) -> Option<&str> {
+    if let Some(result) = storage::DICTIONARIES
+        .iter()
+        .find(|(_, words)| words.contains(input))
```

Comment:
How can we get the word found in the dictionary + the filename from this? :)

Reply:
do you mean word + filename? as the filename is the name of dictionary right 😁